recipes/pytrilinos/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "PyTrilinos" %}
+{% set version = "12.10.1" %}
+{% set sha256 = "ea8ff573bb1f452f2585950cdabe7e2c84cb35013844c58dc878c66bd4245e06" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: trilinos-{{ version }}-Source.tar.gz
+  url: http://trilinos.csbsju.edu/download/files/trilinos-{{ version }}-Source.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - patch.txt
```

Comment:
The patch is needed because my build system was calling a script I had written with the shebang line

    #! ${PYTHON_EXECUTABLE}

which expands out to the 256+ character path for python, but the Linux environment for conda-forge limits the shebang line to 80 characters. The fix was to run the script by specifying the python executable on the command line rather than depending on the shebang. This fix required modifying several `CMakeLists.txt` files.